src/Components/App/App.js

```diff
@@ -70,20 +89,30 @@ const App = () => {
   const filterChosenVendor = () => {
 
   }
+
   const resestResults = (event) => {
     event.preventDefault();
     fetchData()
   }
-const fetchData = () => {
-  const thisData = mockData.data.map((data) => {
-    return data.attributes
-  })
-  setVendors( thisData)
-  // this is going to fetch the data and then set state but then also reset isLoading to false
-}
+
+  const fetchData = () => {
+    const thisData = mockData.data.map((data) => {
+      return data.attributes
+    })
+    setVendors(thisData)
+    // this is going to fetch the data and then set state but then also reset isLoading to false
+  }
+
   useEffect(() => {
-    fetchData();
-// call in mock data somehow and establish it as the state of vendors
+    fetchData()
+    
+    socket.on('receive_data', (data) => {
```

Comment:
It's interesting/makes sense that is also has to receive the message or data before it shows it on the DOM